runtime/src/launch/aprun/aprun-utils.c

```diff
@@ -201,6 +202,22 @@ int getCPUsPerCU() {
   return numCPUsPerCU;
 }
 
+const char *getHostListStr() {
+  return getAprunArgStr(aprun_L);
+}
+char *getHostListOpt() {
+  const char *hostList = getenv("CHPL_HOSTLIST");
```

Comment:
Launcher envs should use the `CHPL_LAUNCHER` prefix (see other getenvs in runtime/src/launch)

slurm-srun uses `CHPL_LAUNCHER_NODELIST`, though I'm not opposed to using `CHPL_LAUNCHER_HOSTLIST` if you prefer. 

Also note that you'll want to update pbs-aprun/aprun-utils.c too (at some point we tried making that file a symlink, but we ran into some limitations that I'm not remembering off the top of my head)

Reply:
Thanks.  I'll change it to look like slurm.

Reply:
Cool, thanks -- it looks like the aprun man page uses the term `node-list`/`node_list` anyways

Reply:
FYI I was curious why we stoped using a symlink -- looks like a few years ago we had some old git clients that couldn't support symlinks correctly or something: https://github.com/chapel-lang/chapel/commit/51ddc2c03cbb6de01bc1bd25f8f90a7055364661 https://github.com/chapel-lang/chapel/commit/9f48237ef1814a9b3dcf9e66b8373067c300b0be